src/components/table/FaultTreeTableBody.tsx

```diff
@@ -29,28 +31,46 @@ const FaultTreeTableBody: FC<FaultTreeTableBodyProps> = ({ faultTrees, handleFau
     navigate(routePath);
   };
 
-  const failureRate = (fr: number) => (
-    <Tooltip title={fr}>
-      <Typography>{fr?.toExponential(2)}</Typography>
-    </Tooltip>
+  const failureRate = (fr: number, icon = null) => (
+    <Stack alignItems="center" direction="row" gap={0}>
+      <Tooltip title={fr}>
+        <Typography>{fr?.toExponential(2)}</Typography>
+      </Tooltip>
+      {!!icon && icon}
+    </Stack>
   );
 
   return (
     <>
       {faultTrees.map((faultTree, rowIndex) => {
         const routePath = ROUTES.FTA + `/${extractFragment(faultTree.iri)}`;
-        const statusColor = faultTree?.status !== Status.OK ? "red" : "black";
+        const isSynced = faultTree?.status === Status.OK;
+        const syncStatusColor = isSynced ? theme.synchronized.color : theme.notSynchronized.color;
+        const violatedRequirement =
+          faultTree?.requiredFailureRate &&
+          faultTree?.calculatedFailureRate &&
+          faultTree?.calculatedFailureRate > faultTree?.requiredFailureRate;
+        const violatedRequirementStatusColor = violatedRequirement ? theme.main.red : theme.main.black;
```

Comment:
@kostobog 
Not sure but don't we want to use custom style here instead saying `red`

Reply:
@blcham 
You mean something like `theme.requirementViolation.color`?

Reply:
Yes something like that ...